cernopendata/modules/fixtures/data/records/cms-tools-eventproduction-examples.json

```diff
@@ -0,0 +1,194 @@
+[
+  {
+    "abstract": {
+      "description": " <p>This is a collection of examples on how to produce working CMSSW python configuration files that can be used to perform the generation, simulation and/or reconstruction of Monte Carlo (MC) (or real) collision events. These events can be used later to perform physics analyses.</p> <p>Each package in this example guides you to produce a few events of the process referenced in the name of the package. The instructions in each allow you to produce the necessary configuration files that are needed and which have been stored in the repository for comparison. We also show you how to execute the relevant jobs.</p> "
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "Carrera, Edgar"
+      },
+      {
+        "name": "Montero, Cesar "
+      }
+    ],
+    "collections": [
+      "CMS-Tools"
+    ],
+    "date_created": [
+      "2010"
+    ],
+    "date_published": "2019",
+    "distribution": {
+      "formats": [
+        "gz"
+      ],
+      "number_files": 1,
+      "size": 9999
+    },
+    "doi": "FIXME",
+    "experiment": "CMS",
+    "files": [
+      {
+        "checksum": "FIXME",
+        "size": 9999,
+        "uri": "FIXME"
+      }
+    ],
+    "license": {
+      "attribution": "GNU General Public License (GPL) version 3"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "FIXME",
+    "run_period": [
+      "Run2010B"
+    ],
+    "system_details": {
+      "description": "Use this code with the CMS Open Data VM environment",
+      "recid": "250",
+      "release": "CMSSW_4_2_8"
+    },
+    "title": "Examples for event generation with 2010 CMSSW machinery",
+    "type": {
+      "primary": "Software",
+      "secondary": [
+        "Tool"
+      ]
+    },
+    "usage": {
+      "description": " <p>If you do not have the CERN Virtual Machine for 2010 CMS data installed, follow the instructions in step 1 at <a href=\"/VM/CMS/2010\">How to install a CERN Virtual Machine</a>.</p>\n\t\t\t<p>To run the examples, follow the instructions in each package at <a href=\"https://github.com/cms-opendata-analyses/EventProductionExamplesTool/tree/2010\">https://github.com/cms-opendata-analyses/EventProductionExamplesTool/tree/2010</a>.</p> "
+    },
+    "use_with": {
+      "description": "Use this with CMS 2010 software environment."
+    }
```

Comment:
`use_with` refers usually to the input data samples (either exact records or something like "any primary dataset containing XXX". In this case there's no input so it can be left out. The software environment is already described in `usage`

Reply:
thanks.